controllers/openstackcluster_controller.go

```diff
@@ -239,40 +223,75 @@ func (r *OpenStackClusterReconciler) reconcileNormal(ctx context.Context, log lo
 		subnetOpts.NetworkID = networkList[0].ID
 		subnetList, err := networkingService.GetSubnetsByFilter(&subnetOpts)
 		if err != nil || len(subnetList) == 0 {
-			return reconcile.Result{}, errors.Errorf("failed to find subnet: %v", err)
+			return errors.Errorf("failed to find subnet: %v", err)
 		}
 		if len(subnetList) > 1 {
-			return reconcile.Result{}, errors.Errorf("failed to find only one subnet (result: %v): %v", subnetList, err)
+			return errors.Errorf("failed to find only one subnet (result: %v): %v", subnetList, err)
 		}
 		openStackCluster.Status.Network.Subnet = &infrav1.Subnet{
 			ID:   subnetList[0].ID,
 			Name: subnetList[0].Name,
 			CIDR: subnetList[0].CIDR,
 		}
+
+		if openStackCluster.Spec.ManagedAPIServerLoadBalancer {
```

Comment:
I think we can just move this part below the if/else so we don't have to duplicate it?

Reply:
Yes, It was basically a copy/paste.